### PR TITLE
update molo version to molo version 10.2 which deprecates reaction qu…

### PR DESCRIPTION
**Pull request by moh-moola**
…estions

### CODE REVIEW

File: gem/tests/test_views.py [hunk 1]
```diff
@@ -626,72 +624,3 @@ def test_extra_params_set(self):
         self.assertEqual(view.get_extra_params(request), {
             'theme': 'springster', 'language': 'en'
         })
-
-
-class ChhaaJaaReactionQuestionsTest(TestCase, GemTestCaseMixin):
```
Comment (KaitCrawford) [inline gem/tests/test_views.py:631]: Please replace this test with one that checks the same things for the new forms reaction questions

File: gem/templates/malawi/core/related-article-list.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
+{% load wagtailcore_tags wagtailimages_tags core_tags %}
+
```
Comment (KaitCrawford) [inline gem/templates/malawi/core/related-article-list.html:2]: Why is this template part of this PR? it doesn't seem to be related to any of this work.

Comment (moh-moola) [reply gem/templates/malawi/core/related-article-list.html:2]: @KaitCrawford not sure as to why this was not causing errors before, but the omitting this template will cause 500 errors

Comment (KaitCrawford) [reply gem/templates/malawi/core/related-article-list.html:2]: @moh-moola was it raising errors on your local instance or in the tests?

Comment (moh-moola) [reply gem/templates/malawi/core/related-article-list.html:2]: @KaitCrawford locally

File: gem/tests/test_views.py (hunk 1 above)
Comment (moh-moola) [reply gem/tests/test_views.py]: @KaitCrawford done

File: gem/tests/test_views.py [hunk 3]
```diff
@@ -637,15 +643,68 @@ def setUp(self):
...
+            )
+
+            res = self.client.get('/')
+            self.assertEqual(res.status_code, 200)
+            self.assertContains(res, article.title)
+            self.assertTrue(form.title in str(res.content))
+            self.assertFalse(form_field.label in str(res.content))
 
-    def test_highlights_correct_reaction(self):
+    def test_reaction_question_multi_submissions(self):
```
Comment (KaitCrawford) [inline gem/tests/test_views.py:700]: I have a number of concerns with the changes to this test. I'll schedule a call with you to discuss it rather than us going back and forth here.

File: test_settings.py [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': 'gem_test.db',
-    }
-}
+# DATABASES = {
+#     'default': {
+#         'ENGINE': 'django.db.backends.sqlite3',
+#         'NAME': 'gem_test.db',
+#     }
+# }
```
Comment (KaitCrawford) [inline test_settings.py:7]: I don't think this should be commented out should it? I'm guessing you did it for your local testing?

Comment (moh-moola) [reply test_settings.py:7]: @KaitCrawford Due to sqlite syntax issues when running migrations, I had to force test settings to use a postgres db

Comment (KaitCrawford) [reply test_settings.py:7]: @moh-moola did you add other db settings to get the migrations running locally? They aren't in this PR

Comment (moh-moola) [reply test_settings.py:7]: @KaitCrawford this is all that was needed

Comment (KaitCrawford) [reply test_settings.py:7]: @moh-moola I'm worried this is going to break things for the rest of us locally.
Because without these settings then the app will still use sqlite because of these settings https://github.com/praekeltfoundation/molo-gem/blob/develop/gem/settings/base.py#L255
Did you maybe set the `DATABASE_URL` setting in your local environment to use postgres?

Comment (moh-moola) [reply test_settings.py:7]: @KaitCrawford Nope, locally and on trav there are no test issues, I removed just the explicit declaration of the sqlite db conf
